tools/runners/run-test-suite-test262.sh

```diff
@@ -14,108 +14,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-TARGET=$1
-shift
+ENGINE="$1"
+REPORT_PREFIX="report"
+RUN_PIDS=""
+RESULT_OK=1
 
-PARSE_ONLY=""
-VERBOSE_REPORT=0
-TIMEOUT=5
+rm "${REPORT_PREFIX}".* &> /dev/null
 
-function usage() {
```

Comment:
What is the current usage of this script? We should write a comment at the beginning of this file at least.